chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
+
+const (
+	RawTx HexFormatFlag = 1
+	Ef    HexFormatFlag = 2
+)
+
+func (flag HexFormatFlag) Contains(other HexFormatFlag) bool {
+	return (flag & other) == other
+}
+
 // Broadcast will attempt to broadcast a transaction using the given providers
-func (c *Client) Broadcast(ctx context.Context, id, txHex string, timeout time.Duration) (string, error) {
+func (c *Client) SupportedBroadcastFormats() HexFormatFlag {
```

Comment:
`SupportedBroadcastFormats` func should have a comment starting with its name. I think that here there is a leftover from `Broadcast` func comment